### PR TITLE
Added dark mode

### PR DESCRIPTION
**Pull request by kailash360**
I have added the dark mode functionality for the web-app.  It would be nice if you can also attach the 'hacktoberfest' label to the issue.

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -3,15 +3,24 @@ import "./App.css";
 import Navigation from "./components/Navigation/index";
 import Favorites from "./pages/Favorites/index";
 import Home from "./pages/Home/index";
+import { ThemeProvider } from 'styled-components';
+import { lightTheme, darkTheme } from './ThemeChanger/themes';
+import { GlobalStyles } from './ThemeChanger/global';
+import {useDarkMode} from './ThemeChanger/useDarkMode'
```
Comment (nishidhaSri) [inline src/App.js:9]: Add proper spacing here

File: src/components/Card/styles.module.css [hunk 2]
```diff
@@ -1,58 +1,63 @@
 .root {
-  width: 350px;
-  height: 500px;
-  margin: 10px;
+    width: 350px;
+    height: 500px;
+    margin: 10px;
```
Comment (nishidhaSri) [inline src/components/Card/styles.module.css:4]: Remove changes not required and adhere to the proper indentation according to rest of the project

File: src/components/Grid/index.jsx [hunk 3]
```diff
@@ -1,22 +1,41 @@
...
 const Grid = ({ view, setView }) => {
   return (
     <div className={styles.gridAlign}>
-      {[z, y, x].map((img, i) => (
+      {/* {[z, y, x].map((img, i) => ( */}
         <a
-          key={i}
-          onClick={() => setView(i + 1)}
-          className={i + 1 === view ? styles.selected : null}
+          key={0}
```
Comment (nishidhaSri) [inline src/components/Grid/index.jsx:12]: Avoid repetition of the code, try to use `.map` for the `ReactComponent`

File: src/components/Grid/styles.module.css [hunk 4]
```diff
@@ -1,24 +1,26 @@
...
-  position: fixed;
-  display: flex;
-  flex-direction: column;
-  top: 40%;
-  right: 50px;
+    position: fixed;
+    display: flex;
+    flex-direction: column;
+    top: 40%;
+    right: 50px;
```
Comment (nishidhaSri) [inline src/components/Grid/styles.module.css:6]: Correct the indentation

**Comment by kailash360** [comment]
I have made the changes as asked, mam

**Comment by nishidhaSri** [comment]
That's great work! Kudos!

**Pull request merged**